Fushigi/Fushigi.csproj

```diff
@@ -18,7 +18,9 @@
     <PackageReference Include="Silk.NET.OpenGL" Version="2.18.0" />
     <PackageReference Include="Silk.NET.Input" Version="2.18.0" />
     <PackageReference Include="Silk.NET.Input.Extensions" Version="2.18.0" />
+    <PackageReference Include="Silk.NET.OpenGL.Extensions.ImGui" Version="2.18.0" />
     <PackageReference Include="StbImageSharp" Version="2.27.13" />
+    <PackageReference Include="ZstdNet" Version="1.4.5" />
```

Comment:
Is there a reason ZstdNet is included? I think the existing ZstdSharp package should be all that we need.

Reply:
I don't remember putting this in to be honest.